hera_cal/smooth_cal.py

```diff
@@ -82,55 +82,96 @@ def dpss_filters(freqs, times, freq_scale=10, time_scale=1800, eigenval_cutoff=1
         time_scale: time scale in seconds. Defined analogously to freq_scale.
             Note that time_scale is in seconds, times is in days.
         eigenval_cutoff: sinc_matrix eigenvalue cutoff to use for included dpss modes.
-            Only used when the filtering method is 'DPSS'
```

Comment:
Why delete this line? It might seem redundant with the advanced user, but I think it's better to be explicit.